problems.rb

```diff
@@ -0,0 +1,9 @@
+#TODO: fix this whole fuction
+def hithisfuncissocomp!!licated(n)
```

Comment:
I love what you're doing here but it seems that the name of the function is a bit too long and not so intuitive. We could replace it with something like sayHiFourTimes(n).